### ISSUE TITLE
Curriculum "Edit Page Contents" button not keyboard navigable in Firefox: CMU-293

### ISSUE THREAD

**Issue by darrensiegel** [Simon-Initiative/oli-torus#516]
This works fine in Chrome, but in Firefox I cannot tab to the "Edit Page Contents" button in the Curriculum view

**Comment by LEDThereBeLight** [comment]
This seems to be a Mac OS issue. See https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox. It's an old answer but the solutions work.

**Comment by LEDThereBeLight** [comment]
Still an issue as of 3/1/22
